erts/emulator/Makefile.in

```
@@ -209,8 +209,7 @@ LIBS += $(TYPE_LIBS)
 ORIG_LIBS:= $(LIBS)
 
 comma:=,
-space:=
-space+=
+space:=$(subst ,, )
```

I find this a little bit confusing. The documentation for `make` suggests the following, which I think is clearer:

```suggestion
empty:=
space:=$(empty) $(empty)
```

---

fixes